assets/js/phoenix_live_view.js

```diff
@@ -556,7 +556,7 @@ export class LiveSocket {
   }
 
   withinTargets(el, phxTarget, callback){
-    if(/^(0|[1-9](\d?)+)$/.test(phxTarget)){
+    if(/^(0|[1-9]\d*)$/.test(phxTarget)){
```

Comment:
The resource here is pretty cheap, but we can save re-instantiating RegExp objects on every check by extracting this as a separate static member of this class.

Reply:
I think I'll skip that in this PR to keep it focused to one thing, but fair point :)